docs/guides/src/main/server/containers.adoc

```diff
@@ -25,15 +28,15 @@ ENV KC_HEALTH_ENABLED=true
 ENV KC_METRICS_ENABLED=true
 ENV KC_FEATURES=token-exchange
 ENV KC_DB=postgres
+# Install custom providers
+RUN curl -sL https://github.com/aerogear/keycloak-metrics-spi/releases/download/2.5.3/keycloak-metrics-spi-2.5.3.jar -o /opt/keycloak/providers/keycloak-metrics-spi-2.5.3.jar
 RUN /opt/keycloak/bin/kc.sh build
 
 FROM quay.io/keycloak/keycloak:latest
-COPY --from=builder /opt/keycloak/lib/quarkus/ /opt/keycloak/lib/quarkus/
+COPY --from=builder /opt/keycloak/ /opt/keycloak/
```

Comment:
Is it necessary to copy full /opt/keycloak ? Can't we just do this instead?
```
COPY --from=builder /opt/keycloak/lib/quarkus/ /opt/keycloak/lib/quarkus/
COPY --from=builder /opt/keycloak/providers/ /opt/keycloak/providers/
```

Reply:
I get about ~200MB lighter image if I follow this approach instead of fully copying /opt/keycloak/l

Reply:
@andreaTP any answer to that question? Would also be interested in it.

Reply:
Cc. @pedroigor

Reply:
@armujahid It should work fine because files at `lib/quarkus` are what really changes after a `build`.

If you want to send a PR with this suggestion to our guide, I'm happy to review it and make this more explicit.